sandbox/helpers/related-person-helper.js

```diff
@@ -0,0 +1,90 @@
+const Boom = require('boom')
+const datefns = require('date-fns')
+const relatedpersons = require('../services/relatedpersons')
+
+function _getRelatedPersons(nhsNumber) {
+    let response = [];
+        Object.keys(relatedpersons).forEach(key => {
+            if (nhsNumber === key) {
+                Object.keys(relatedpersons[key]).forEach(responseKey => {
+                    response.push(relatedpersons[key][responseKey]);
+                })
+            }
+        })
+
+        if (response.length === 0) {
+            throw Boom.notFound(
+                `Resource not found`,
+                {operationOutcomeCode: "not_found", apiErrorCode: "RESOURCE_NOT_FOUND"}
+            )
+        }
+
+        return response
+}
+
+function buildBundleResponse(exampleResources = [], nhsNumber) {
+    let response = {
+        resourceType: "Bundle",
+        type: "searchset",
+        timestamp: datefns.format(Date.now(), "yyyy-MM-dd'T'HH:mm:ss+00:00"),
+        total: exampleResources.length,
+        entry: []
+    }
+
+    if (exampleResources.length > 0) {
+        exampleResources.forEach(resource => {
+            response.entry.push({
+                fullUrl: "https://beta.api.digital.nhs.uk/Patient/" + nhsNumber + "/RelatedPerson/" + resource["id"],
+                resource: resource,
+            })
+        });
+    }
+    return response;
+}
+
+module.exports = {
+    /**
+     * Used by Patient/{nhsNumber}/RelatedPerson paths to check it there are any
+     * Related Person resources attached to the patients nhsNumber.
+     *
```

Comment:
patient's
I don't know whether this should be "NHS Number", rather than the variable name of nhsNumber. Need to think about this...